src/Kuzzle.js

```diff
@@ -210,6 +197,8 @@ function Kuzzle (host, options, cb) {
     }
   }
 
+  this.eventEmitter = new EventEmitter(this.eventTimeout);
```

Comment:
Do we need a property to handle events? Wouldn't it make more sense if the main object (`Kuzzle` || `WSNode`) would be an event emitter itself?